src/Search.css

```diff
@@ -0,0 +1,64 @@
+@import url('https://fonts.googleapis.com/css?family=Itim');
+
+input[type=search]{
+   -moz-appearance: none;
+   -webkit-appearance: none;
+   appearance: none;
+}
+
+.search-div {
+  height: 50px;
+}
+
+.search-bar {
+  width: 40%;
```

Comment:
You'll want to make sure your declarations within each declaration block are organized in some way - either by grouping like properties ([see idiomatic css](https://github.com/necolas/idiomatic-css)) or listing declarations alphabetically.